tunnel-server/src/auth.ts

```diff
@@ -42,15 +44,13 @@ export type BearerAuthorizationHeader = {
 }
 
 export type AuthorizationHeader = BasicAuthorizationHeader | BearerAuthorizationHeader
-
-const cookieName = 'preevy-saas-jwt'
-
-type VerificationData = {
-  pk: KeyObject
-  extractClaims: (token: JWTPayload, thumbprint: string) => Claims
+export type JWTIssuer = {
```

Comment:
the term `issuer` is used ambiguously in this flow for a name (string), object, payload - IMO need to replace with more specific terms.

Reply:
I can use IdentityProvider

Reply:
Updated it.